pkg/resources/pullSecret.go

```diff
@@ -54,3 +57,25 @@ func CopySecret(ctx context.Context, client k8sclient.Client, srcName, srcNamesp
 
 	return err
 }
+
+// Copies a secret from the RHMI operator namespace to a target namespace
+func ReconcileSecretToProductNamespace(ctx context.Context, client k8sclient.Client, configManager config.ConfigReadWriter, secretName string, namespace string) (integreatlyv1alpha1.StatusPhase, error) {
+	err := CopySecret(ctx, client, secretName, configManager.GetOperatorNamespace(), secretName, namespace)
+
+	if err != nil && !k8serr.IsNotFound(err) {
+		return integreatlyv1alpha1.PhaseFailed, fmt.Errorf("failed to seed copy secret to product namespace: %w", err)
+	}
+
+	return integreatlyv1alpha1.PhaseCompleted, nil
```

Comment:
@KevFan This will return complete even if the secret doesn't exist which it won't on the first try. I don't think it needs to be changed for this PR since this needs to get merged quickly. I would prefer to see a non-blocking failure  or even a log here in this case as it might be hard to debug an issue in the future if there is an issue caused because of this case.

Reply:
Yup, sure sounds good to me :+1: